packages/storybook/src/example-pages/toepassingen/aanvragen/locatie.ts

```diff
@@ -0,0 +1,102 @@
+import { html } from "lit-html";
+
+import { examplePageFactory } from "../../../example-page-factory";
+import { dropdownItems } from "./locatie.content";
+
+examplePageFactory(
+  "Toepassingen/Aanvragen",
+  "Locatie",
+  ({
+    applicationHeadingTemplate,
+    dropdownMenuTemplate,
+    formGroupSelectTemplate,
+    formGroupInputTemplate,
+    formButtonsTemplate,
+    buttonTemplate,
+  }) => html`
+    <style>
+      .dso-map-example {
+        background-image: url("/images/map-lved125.png");
+        background-size: cover;
+        height: 400px;
+        margin: 32px 0;
+        width: 100%;
+      }
+    </style>
+
+    <div class="container">
+      <main>
+        ${applicationHeadingTemplate({
+          title: "Aanvraag Laan van Eik en Duinen 125, Den Haag",
+          subtitle: "2. Locatie",
+          step: "Stap 2/7",
+        })}
+        <form>
+          <div class="row">
+            <div class="col-md-9">
+              <h3>Coordinaten</h3>
+            </div>
+            <div class="col-md-3 text-right">
+              ${dropdownMenuTemplate({
+                button: { variant: "tertiary", label: "Meer zoekopties" },
+                id: "locatie--dropdownmenu",
+                groups: dropdownItems,
+              })}
+              <!-- {% render '@dropdown-menu-css', dropdownButton %} -->
+            </div>
+          </div>
+          <div class="dso-justify-form-groups">
```

Comment:
Dit is een component toch?

Reply:
Goed gezien, die heb ik gemist. Even kijken hoe ik dit kan samenvouwen